src/drivers/intel/cavs/Kconfig

```diff
@@ -92,6 +93,13 @@ config CAVS_DMIC_FIR_DECIMATE_BY_8
 	  This includes FIR coefficients to decimate by 8
 	  into the build.
 
+config CAVS_DMIC_FIR_DECIMATE_BY_12
+	bool "FIR decimate by 12"
+	default n
+	help
+	  This includes FIR coefficients to decimate by 12
```

Comment:
Best explain what this and the other decimate value are used for. i.e list rates/formats that each decimate value is used for.

Reply:
There's no direct relation with FIR decimation factors and supported sample rates. The driver is less likely to reject a configuration request for some PDM bus clock range, duty cycle range, sample rate the more FIR decimation factors are enabled in kconfig.

The high factors like 12 get used for the low rate (16 kHz) output if there's a request for a high rate (96 kHz) output.  

The smallest decimation factor is 2. If the 96 kHz output is decimated by 2 from 192 kHz, then the 16 kHz output is decimated by 12 from 192 kHz.

Reply:
@singalsu ok thanks, this is useful info that could be on the help menu for decimation.

Reply:
Yep, I'll do it for next version of this PR. I got also information from @lbetlej that need for this feature is postponed. We should decide if this should be left out from default builds for now but kept available for quick add if it would be needed later.

This was also a good case for the configuration tool to generate coefficients. I need to make a PR to add it to SOF/tools as well.